provider/cmd/pulumi-gen-eks/main.go

```diff
@@ -1063,17 +1063,13 @@ func generateSchema() schema.PackageSpec {
 					"eksCluster",
 				},
 			},
-			// The TypeScript library exports this at the top-level index.ts, so we expose it here. Although, it's not
-			// super useful on its own, so we could consider *not* exposing it to the other languages.
-			// Note that this is a _custom_ resource (not a component), implemented in the provider plugin.
-			// Previously it was implemented as a dynamic provider.
-			"eks:index:VpcCni": {
+			"eks:index:VpcCniAddon": {
```

Comment:
So this https://www.pulumi.com/registry/packages/eks/api-docs/vpccni/ is removed in favor of a new one it seems, ack.